benchmarks/gravity_thin_shell/gravity_thin_shell.prm

```diff
@@ -0,0 +1,83 @@
+# A simple setup for testing the gravity postprocessor using the 
+# sample scheme 'map'. The domain is a 5km thick thin shell of constant density.
+# An analytical solution exists for the gravitational acceleration 
+# and potential so that values returned by the gravity postprocessor
+# can be benchmarked on a longitude-latitude grid at satellite orbit height.  
+
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output_thin_shell_gravity
+set Nonlinear solver scheme                = no Advection, no Stokes
+
+# Model geometry
+# The thin shell (5km thickness) can be centered around 5 different depths, 
```

Comment:
Could you add a space between the number and "km" for this and the following lines (also in the Spherical shell subsection)?

Reply:
done

Reply:
also indent